content/docs/1.1.1/references/settings.md

```diff
@@ -43,10 +43,11 @@ weight: 1
   - [Storage Over Provisioning Percentage](#storage-over-provisioning-percentage)
 - [Danger Zone](#danger-zone)
   - [Disable Replica Rebuild](#disable-replica-rebuild)
-  - [Guaranteed Engine CPU](#guaranteed-engine-cpu)
+  - [(Deprecated)Guaranteed Engine CPU](#deprecatedguaranteed-engine-cpu)
```

Comment:
Move deprecated into another section.

Reply:
Agree, we should probably have a dedicated deprecated section.

Reply:
Updated